File: 프로그래머스/lv1/42576. 완주하지 못한 선수/완주하지 못한 선수.py
def solution(participant, completion):
    hash = {}
    for person in participant:
        if person in hash:
            hash[person] +=1
        else:
            hash[person] = 1
    
    for person in completion:
        if person in hash:
            hash[person] +=1
        else:
            hash[person] = 1
    for key in hash:
        if hash[key] % 2 != 0:
            return key

File: 프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.js
function solution(participant, completion) {
    const count = new Map()
    
    for (const person of participant) {
        if (count.has(person)){
            count.set(person, count.get(person) + 1)
        } else {
            count.set(person, 1)
        }
    }
    
    for (const person of completion) {
        count.set(person, count.get(person) - 1)
    }
    
    
    for (const [key, value] of count) {
        if (value > 0) {
            return key
        }
    }
}